analytics/location_heatmaps/geo_utils.py

```diff
@@ -499,18 +500,11 @@ def convert_to_dataset(image, total_size, value=None):
 
 
 def compute_conf_intervals(sum_vector: np.ndarray, level=95):
+  from scipy.stats import norm
```

Comment:
I'd prefer not to do imports inline like this.

Reply:
done